### DISCUSSION TITLE
Image field in directus gets created as string?

### DISCUSSION THREAD

**Discussion by gardemanuelsen** [directus/directus#16253] [Archive]
### Describe the Bug

Hi!

I´am a student and new to Directus. I have a problem displaying images from directus to my database. I´am able to display all fields that are created as input, but image fields seems like they are being created as strings, so i cannot access any id on the image. This i s my output when I do a query for the image.
<img width="1643" alt="Screenshot 2022-11-01 at 16 08 02" src="https://user-images.githubusercontent.com/74247742/199266639-9389dd58-88fb-4bcb-8fce-f5a69790a74b.png">

My teacher has the option to select id in the image field, I think this is the right behaviour
<img width="1643" alt="Screenshot 2022-11-01 at 16 08 02" src="https://user-images.githubusercontent.com/74247742/199267341-8ee8d7ab-bca2-479d-8ee2-ffd0271335ce.png">

My question is if there are something I´am doing wrong when I create a image field or is it a bug with my directus project

Thanks in advance!

### To Reproduce

:)

### Errors Shown

<img width="1676" alt="Screenshot 2022-11-01 at 16 12 35" src="https://user-images.githubusercontent.com/74247742/199267631-37c6329e-de4e-4908-aa36-364ddc8a2657.png">

### What version of Directus are you using?

9.18.1

### What version of Node.js are you using?

16.17.0

### What database are you using?

GraphQL

### What browser are you using?

Chrome

### How are you deploying Directus?

locally

**Comment by br41nslug** [comment]
Image fields are strings because they are a relation to the `directus_files` collection and contain the related primaryKey which is an UUID.
To render this in your app you can use `/assets/:UUID:` (with access token if not publicly accessible) https://docs.directus.io/reference/files.html#accessing-a-file

Edit: Sorry seems that is what you're already doing 😬 As you're getting a Forbidden exception are you sure the token is still valid and has access to files?

**Reply by gardemanuelsen** [reply]
ah okay, I see!

I think the setup is correct. Is there anyway I can make sure. This how the user settings are right now:
<img width="1680" alt="Screenshot 2022-11-02 at 14 04 34" src="https://user-images.githubusercontent.com/74247742/199496866-89334ba4-d4a7-42b2-bd9f-84d24d594c67.png">

**Reply by br41nslug** [reply]
Upon revisiting this question i realized this may be caused by the fact that system tables (including users/files) have a separated endpoint for querying. https://docs.directus.io/reference/system/fields.html#graphql-2

So it may not be possible to expand into these kinds of relations in one call.

**Reply by gardemanuelsen** [reply]
I managed to fix the problem. I remade the User directory, and then I was able to select ID as subfield of img. Thanks for your help!

**Comment by sle78** [comment]
I had the same issue and the reason was no permissions had been granted on fields. 
This is what worked for me:
<img width="834" alt="image" src="https://github.com/user-attachments/assets/eb3cd0a7-e3d2-491b-b615-3c79c3569565" />
